linode/manager
# M3-6175: Update Cloud Manager to Node.js 18.14.1 LTS

## Description 📝

- Updates Cloud Manager's Node.js version to the current latest LTS version
  - Currently we are on `14.17.4` and we will update to the latest LTS version which is `18.14.0`
- This update will unblock the ability to use Vite

## How to test 🧪

- Use nvm or volta to switch node version to 18.14.0
  - For me, I ran `volta install node` in the repo's root directory
- Verify Storybook still works
- Verify Github actions CI passes
- Verify internal build works
